plaidml/bridge/openvino/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ if(NOT InferenceEngineDeveloperPackage_FOUND)
   FetchContent_Declare(
     openvino
     GIT_REPOSITORY https://github.com/plaidml/openvino.git
-    GIT_TAG        04750e225e633fd07ae28b2bf28c3e8202ad7c2b
+    GIT_TAG        ea65cfebb27b5fd9be8bb3c602746a79d03d5b9d
```

Comment:
This will need to be updated with the hash of the now-landed commit

Reply:
ok, fix this